app/Http/Livewire/Components/Header.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Livewire\Components;
+
+use App\Http\Livewire\Traits\InteractWithModal;
+use App\Models\Configs;
+use Illuminate\Support\Facades\Config;
+use Livewire\Component;
+
+class Header extends Component
```

Comment:
Please add a PHPdoc comment what this class does and how it relates to other classes. It is the header of what?